CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Fixed missing word for plugin installation in README @budhrg
 - Fix links, typos, formatting in CONTRIBUTING.md @budhrg
 - Fix #16 and #72: Enable private networking for VirtualBox if not set @budhrg
+- Remove manually scp for TLS keys and use machine.communicate.download @bexelbie
```

Comment:
We should put the new changelogs at the top. Fine for now, keep a note henceforth.